pytest.ini

```diff
@@ -17,13 +17,14 @@
 [pytest]
 addopts = --verbose
     --doctest-modules
+    -n=1
+    --dist=loadscope
     --ignore=cylc/flow/parsec/empysupport.py
     --ignore=cylc/flow/parsec/validate.py
     --ignore=cylc/flow/parsec/example
-    --ignore=cylc/flow/tests/parsec/getcfg/bin/one-line.py
-    --ignore=cylc/flow/tests/parsec/synonyms/bin/synonyms.py
-    --ignore=cylc/flow/tests/parsec/nullcfg/bin/empty.py
+    --ignore=tests/unit/parsec/getcfg/bin/one-line.py
+    --ignore=tests/unit/parsec/synonyms/bin/synonyms.py
+    --ignore=tests/unit/parsec/nullcfg/bin/empty.py
     --ignore=cylc/flow/data_messages_pb2.py
 testpaths =
-    cylc/flow/
-    tests/lib/python/
+    tests/unit/
```

Comment:
```suggestion
    tests/unit/
    tests/integration/
```

Reply:
I purposefully left the integration tests out of `testpaths` so that when you run `pytest` it defaults to running `tests/unit`.

Reply:
Ah, no worries :)